index.js

```diff
@@ -45,13 +49,14 @@ module.exports = async (input, {glob = true, ...options} = {}) => {
 	}
 
 	const filePaths = glob ? await globby(input, {onlyFiles: true}) : input;
+	const baseDirectory = path.dirname(filePaths[0]);
```

Comment:
Why does the first file path decide the base directory? This should be documented.